docs/incubation/graduated.md

```diff
@@ -0,0 +1,12 @@
+---
+weight: 400
+---
+# Advancing to the Graduated stage
+
+The Graduated stage is the most advanced and mature stage of the incubation process. This is for projects which have proven to have a great quality and adoption. These are the lighthouse projects the OpenRail Association recommends for production use throughout the industry.
+
+Projects on the Graduated stage are required to have adopted these practices:
```

Comment:
While I like this short overview, I am also a bit sceptical as it duplicates the information from the incubation process, while many of the points raised there are not listed here.

Reply:
I share the concern. Let's figure that out later, when we have finished the full definition of the incubation process and have decided on the criteria for stage 3. I think we also still have a naming discussion to do as the names of the stages are not ideal. And we might also want to bring up the question again, if the formal definition of the incubation stages could maybe just be part of the project handbook.

Deduplication should not be a big thing, if we decide to do so.